contract/build.gradle.kts

```diff
@@ -0,0 +1,39 @@
+plugins {
+    id(Plugins.AndroidLib.plugin)
+    id(Plugins.Kotlin.plugin)
+    id(Plugins.Parcelize.plugin)
+}
+
+android {
+    namespace = "band.effective.office.contract"
+    compileSdk = 33
+
+    defaultConfig {
+        minSdk = 26
+        targetSdk = 33
+    }
+    sourceSets["main"].apply {
+        manifest.srcFile("src/androidMain/AndroidManifest.xml")
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+
+}
+
+kotlin {
+    android {
+        compilations.all {
+            kotlinOptions {
+                jvmTarget = "1.8"
+            }
+        }
+    }
+    dependencies{
```

Comment:
please, use `sourceSet`, because for iOS, `ktor` needed any dependencies

Reply:
I replace dependencies on sourseSets, but ios set not add, like mocks for mobile